### ISSUE TITLE
[Feature Request] [RetroAchievements] Differentiate missable achievements [m] on UI

### ISSUE THREAD

**Issue by RenanSD007** [hrydgard/ppsspp#19632]
### What should happen

At the moment it is impossible to know if an achievement is [m] through the emulator, in RetroArch for example if an achievement is missable,"[m]" appears before the text "Locked" if the achievement has not been earned, It would be nice to have a different background color for missable achievements while they are in the "Locked" state, when they are unlocked it would not need to differentiate. For example the achievement "Val-i-Klar, the Ashen Turtle!" is missable [m]
![Screenshot_2024-11-13-15-55-42-693_org ppsspp ppsspplegacy](https://github.com/user-attachments/assets/424e7982-b25b-4dc0-9b37-7c245b77b644)

### Who would this benefit

People who just want to use the emulator so they don't have to go to the website to know this information 

### Platform (if relevant)

None

### Games this would be useful in

Games with many missable achievements (mostly RPGs)

### Other emulators or software with a similar feature

RetroArch
![Screenshot_2024-11-13-15-59-01-699_com retroarch](https://github.com/user-attachments/assets/17349d7e-26d3-43fa-9356-f198af027102)

### Checklist

- [X] Check the [latest git build](https://www.ppsspp.org/devbuilds) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by hrydgard** [comment]
#20241 fixes this, but it's not super pretty.

Any better visual ideas?

**Comment by RenanSD007** [comment]
I'm not the best person to give feedback on UI xD, so just the text itself would help a lot, just like in RetroArch, but maybe to make it more beautiful, put icons similar to the ones on the website for "Progression* "Win Condition" "Missable" next to the points?

![missable.png](https://github.com/user-attachments/assets/8ff9bdc6-5336-4b28-9870-748f8a363a2e)

![win.png](https://github.com/user-attachments/assets/04fb50b6-c48a-40a2-bf6e-b98a963f3553)

![progression.png](https://github.com/user-attachments/assets/aeacfd4d-4da6-44a0-8302-606603e6a719)

**Comment by hrydgard** [comment]
What does "progression" even mean? main story?

Anyway, I'll re-tag the issue for 1.20 to prettify it up then.

**Comment by RenanSD007** [comment]
Yes, "Progression" means all the achievements related to the main story or all the levels you need to complete a platform game (for example SMW only need Castle 1, Star World 1,2,3,4 and Bowser Castle to actually beat it)

**Comment by hrydgard** [comment]
I've made some text badges, prettier than [m] etc, for now. Might add graphical badges later, but this should be good enough.

<img width="722" height="332" alt="Image" src="https://github.com/user-attachments/assets/6eeaf090-6e0d-44bd-9a63-ff9f416f5010" />
